recipes-ros/mavlink/mavlink_2016.5.20.bb

```diff
@@ -0,0 +1,26 @@
+#
+# MAVLink message marshaling library.
+# This package provides C-headers and pymavlink library
+# for both 1.0 and 2.0 versions of protocol.
+#
+
+DESCRIPTION = "MAVLink message marshaling library"
+DEPENDS = ""
+LICENSE = "LGPLv3"
```

Comment:
@kubark42: The licensing is clearly stated. The meta data in this repository is under MIT License, the license string indicates the license of the software. Again, this license string is only a very weak indication, not legally relevant information (ultimately, the license information delivered with the software is legally binding). Any patch to the recipe is provided under the license in the recipe to simplify the legal situation when including a patch in the upstream repository with a collaborative effort.